### PR TITLE
validate metadata

### PR DESCRIPTION
**Pull request by reetika-vyas**

### CODE REVIEW

File: hack/yamltojson.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
-import yaml
-import json
+import sys, yaml, json, os
 
-with open("scripts/SREP/example/metadata.yaml", 'r') as yaml_in, open("scripts/SREP/example/metadata.json", "w") as json_out:
-    yaml_object = yaml.safe_load(yaml_in) 
+with open(sys.argv[1], 'r') as yaml_in, open(''.join((sys.argv[1].split('.')[:-1])) + ".json", "w") as json_out:
```
[the remark this answers is not in the document]
Comment (qhua948) [reply hack/yamltojson.py:3]: @georgettica Sorry what do you mean by that?

[the remark this answers is not in the document]
Comment (qhua948) [reply hack/yamltojson.py:3]: The output of the script will be immediately consumed by the next line of script, so I don't think it matters that much given it is an opaque step

File: hack/app_sre_pr_check.sh [hunk 2]
```diff
@@ -4,4 +4,10 @@ set -ex
 
 cd $(dirname $0)/..
 
+#convert yaml to json
+find . -name 'metadata.yaml' -exec python3 hack/yamltojson.py scripts/SREP/example/metadata.yaml  {} \;
```
Comment (wanghaoran1988) [inline hack/app_sre_pr_check.sh:8]: why you put "scripts/SREP/example/metadata.yaml" here?

Comment (reetika-vyas) [reply hack/app_sre_pr_check.sh:8]: I am passing it as argument

Comment (qhua948) [reply hack/app_sre_pr_check.sh:8]: You need to remove that `scripts/SREP/example/metadata.yaml`

Comment (reetika-vyas) [reply hack/app_sre_pr_check.sh]: removed

**Comment by wanghaoran1988** [comment]
```
13:57:01 Traceback (most recent call last):
13:57:01   File "hack/yamltojson.py", line 3, in <module>
13:57:01     with open(sys.argv[1], 'r') as yaml_in, open(''.join((sys.argv[1].split('.')[:-1])) + ".json", "w") as json_out:
13:57:01 FileNotFoundError: [Errno 2] No such file or directory: '/scripts/SREP/example/metadata.json'
13:57:01 + find . -name file:///metadata.json -exec jsonschema --instance '{}' file:///hack/metadata.schema.json ';'
13:57:01 + make IMAGE_REPOSITORY=app-sre build
```
The script failed, but it's not failing the job.

**Pull request closed without merging**